daml/Marketplace/ExchangeParticipant.daml

```diff
@@ -160,3 +160,20 @@ template ExchangeParticipant
           orderRequestCid <- create OrderRequest with ..
 
           return (orderRequestCid, debitRequestCid)
+
+      nonconsuming ExchangeParticipant_MakeClearedOrder : ContractId ClearedOrderRequest
+        with
+          ccp : Party
```

Comment:
Where should the CCP party come from? For now I just made it a parameter of the choice, and on the UI it's currently hardcoded to `"Ccp"` when the choice is exercised. 

If we make it a dropdown on the UI that lists the CCPs that the investor is a customer of, then different investors might enter different CCPs for the same (cleared) market pair, which seems... wrong

Reply:
Hmmm that's a good point. I'll have to ask Eric about customers having multiple CCPs, but I'm guessing that each cleared market will have one CCP attached to it. 

Come to think of it, the CCP already needs to know who the exchange is in order to request `FairValue`s, so I think we'll have to add a specific CCP <-> Exchange relationship.